src/routes/reference/rendering/hydrate.mdx

```diff
@@ -13,3 +13,17 @@ When initializing in the browser a page has already been server rendered.
 ```ts
 const dispose = hydrate(App, document.getElementById("app"))
 ```
+
+## Reference
+
+### `hydrate(fn, node)`
+
+#### Properties
```

Comment:
```suggestion
## Properties
```

Reply:
That's a lot nicer - I'll go through it all